ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -752,8 +797,9 @@ def generate_from_vlan(self):
     def generate_ping_dut_lo(self):
         self.ping_dut_packets = []
         dut_lo_ipv4 = self.lo_prefix.split('/')[0]
-        for src_port in self.vlan_host_ping_map:
-            src_addr = random.choice(self.vlan_host_ping_map[src_port].keys())
+        
+        for src_port in self.active_port_indices if self.test_params['is_dualtor'] else self.vlan_host_ping_map:
+            src_addr = list(self.vlan_host_ping_map[src_port].keys())[0]
```

Comment:
Why always restrict the src_addr to be first vlan host? Randomization here was to generate/mimic ping traffic from different hosts.

Reply:
That was a mistake. Changed back to randomization